packages/cactus-plugin-persistence-fabric/package.json

```diff
@@ -0,0 +1,108 @@
+{
+  "name": "@hyperledger/cactus-plugin-persistence-fabric",
+  "version": "1.2.0",
+  "description": "A web service plugin that provides management capabilities on a Cactus Plugin Fabric Block Persistance as a scrapper of blocks.",
+  "keywords": [
+    "Hyperledger",
+    "Cactus",
+    "Integration",
+    "Blockchain",
+    "Distributed Ledger Technology"
+  ],
+  "homepage": "https://github.com/hyperledger/cactus#readme",
+  "bugs": {
+    "url": "https://github.com/hyperledger/cactus/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/hyperledger/cactus.git"
+  },
+  "license": "Apache-2.0",
+  "author": {
+    "name": "Hyperledger Cactus Contributors",
+    "email": "cactus@lists.hyperledger.org",
```

Comment:
I believe this is now "cacti@lists.hyperledger.org". @petermetz Is that right?

Reply:
> I believe this is now "[cacti@lists.hyperledger.org](mailto:cacti@lists.hyperledger.org)". @petermetz Is that right?

@VRamakrishna Yes, that is correct.

Reply:
> I believe this is now "[cacti@lists.hyperledger.org](mailto:cacti@lists.hyperledger.org)". @petermetz Is that right?

fixed